src/components/v2/V2Project/V2FundingCycleSection/modals/WithdrawModal.tsx

```diff
@@ -0,0 +1,247 @@
+import { Space } from 'antd'
```

Comment:
v2 distribute payouts modal. might rename file later.
Think this warranted a v2 file despite duplication, because theres a lot of curency stuff here and value scaling that I didn't want to mess up.

Reply:
Yeah good call I think. Worth separating if ever in doubt